hydpy/models/hland_v1.py

```diff
@@ -16,6 +16,7 @@
  * Use a submodel like |evap_aet_hbv96| for calculating actual evapotranspiration
    (which, for best compatibility with :cite:t:`ref-Lindstrom1997HBV96`, could use
    |evap_pet_hbv96| for estimating potential evapotranspiration).
+ * Use a optional submodel like |rconc_uh| for calculating runoff concentration.
```

Comment:
"an optional".  However, I wonder if this explains it well.  Maybe "Optionally, use a submodel..."?

Also, please move this point to the end of the list (the list order should roughly reflect the process order).

Reply:
ok